lib/bundled_gems.rb

```diff
@@ -0,0 +1,18 @@
+module Gem
```

Comment:
Why is this duplicated with the `gems/bundled_gems`?

Reply:
I have no plan to handle all of bundled gems like `debug`, `rbs` that are current bundled gems.